build.sbt

```diff
@@ -0,0 +1,171 @@
+organization in ThisBuild := "io.gloriousfuture"
+organizationName in ThisBuild := "The Glorious Future"
+
+version in ThisBuild := "0.0.1"
+scalaVersion := "2.11.8"
+
+licenses in ThisBuild += ("Apache-2.0", url("http://opensource.org/licenses/apache-2.0"))
+
+lazy val root = Project("genrpc-root", file("."))
+  .aggregate(
+    // RPC client code generators
+    `genrpc-core`,
+    // TODO: Move these projects out into a "genspec" repo
```

Comment:
I started doing this, but I starting to question if it is necessary. It's kinda nice to have it all in one repo for now. I just need to be careful to not cross wires when it comes to what depends on what.